aisee/utils.py

```diff
@@ -180,3 +181,32 @@ def check_multilabel_df(df):
             raise ValueError(
                 f"Column '{col}' has values less than 0, values must be between 0 and 1.",
             )
+
+
+def numpy_image_from_jpg(
```

Comment:
No me parece mal meterlo aquí, faltaría el test

Reply:
Done in test_visionclassifier